students/joshua-bone/lesson2/fizz-buzz.py

```diff
@@ -0,0 +1,13 @@
+#Joshua Bone - UW Python 210 - Lesson 2
+#FizzBuzz Exercise
+
+def fizzBuzz():
+  for i in range(1, 101):
+    if i % 3 != 0 and i % 5 != 0: print(i, end='') 
+    if i % 3 == 0: print("Fizz", end='') 
+    if i % 5 == 0: print("Buzz", end='') 
+    print()
```

Comment:
the indentation is incorrect; standard indentation is exactly 4 spaces (no tabs)